airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -223,7 +223,7 @@ def execute(self, context: Context):
                 timeout=self.execution_timeout,
                 trigger=BatchJobTrigger(
                     job_id=self.job_id,
-                    waiter_max_attempts=self.max_retries or 10,
+                    waiter_max_attempts=self.max_retries or 4200,
```

Comment:
Can we not have it as a magic number?
It already has definition in the hook and now separately here.

Reply:
We could change it to be the default in the constructor, so it's more transparent to users? The reason I got tripped up in the linked issue is that it's implied that None is filled in with 4200, but it isn't when `deferrable=True`. 

```
        max_retries: int | None = None,
```